packages/product-feed-heureka/src/Form/HeurekaProductFormType.php

```diff
@@ -38,12 +40,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'currency' => 'CZK',
                 'scale' => 2,
                 'constraints' => [
-                    new Range([
-                        'min' => 0,
-                        'max' => 100,
+                    new MoneyRange([
+                        'min' => Money::fromInteger(1),
+                        'max' => Money::fromInteger(500),
                     ]),
                 ],
             ],
         ]);
+
+        foreach ($builder->get('cpc')->all() as $price) {
```

Comment:
I think you can assign result of `->add` above to a variable like
`$cpc = $builder->add('cpc', MultidomainType::class, [`

and here use `foreach ($cpc->all()...`

Reply:
No, the `add` method returns the parent builder, not the currently added one (so you can call multiple `add`s).